scanomatic/server/scanning_effector.py

```diff
@@ -975,7 +977,7 @@ def _scan_thread(self):
         )
 
     def _setup_directory(self):
-        os.makedirs(self._project_directory)
+        os.makedirs(self._project_directory, exist_ok=True)
```

Comment:
this is actually not really that good, I think there's a point to make that when starting a project it must be a new directory

Reply:
There is something else creating that directory, it always exists and prohibits scanning. One could probably remove _setup_directory...

Reply:
if it always exists then rather remove it if it isn't needed.

Reply:
The mechanism that creates the project dir is the logger. So it's alright to remove the `_setup_directory` function, but it really not a good idea. When/if we replace/change the logger noone creates the project dir. It is probably safe to keep `exists_ok=True` since the experiment-page won't let you to continue if the dir already exists.